packages/types/VideoApiAdapter.d.ts

```diff
@@ -41,6 +42,8 @@ export type VideoApiAdapter =
       ): Promise<TGetTranscriptAccessLink["transcription"] | { message: string }>;
 
       checkIfRoomNameMatchesInRecording?(roomName: string, recordingId: string): Promise<boolean>;
+
+      getWebhooks?(): Promise<TGetDailyWebhooks>;
```

Comment:
It should be a generic schema like `TGetWebhook` as this is a type specification for all apps.